### PR TITLE
F5 LTM Module

### PR DESCRIPTION
**Pull request by cr41gc**
Hey @brian-brazil, I added an F5 LTM module for my own use, and thought I'd share it back. Let me know if you need me to change anything.

### CODE REVIEW

**Comment by cr41gc** [comment]
I just noticed the A10 LB PR that was closed due to the MIBs not being published. F5 handles their MIBs exactly the same as A10 where you're supposed to download them from the device.

Is it good enough that the MIBs are available in the Libre repo?

**Comment by cr41gc** [comment]
I also noticed that some names are coming across has hex strings:

```
# HELP ltmVirtualServStatClientBytesIn 
# TYPE ltmVirtualServStatClientBytesIn counter
ltmVirtualServStatClientBytesIn{ltmVirtualServStatName="0x2F436F6D6D6F6E2F54696D536564657262657267"} 5.784634e+06
ltmVirtualServStatClientBytesIn{ltmVirtualServStatName="0x2F436F6D6D6F6E2F63726169672D74657374"} 39159
ltmVirtualServStatClientBytesIn{ltmVirtualServStatName="0x2F436F6D6D6F6E2F6A6F7368722D74657374"} 197721
ltmVirtualServStatClientBytesIn{ltmVirtualServStatName="0x2F436F6D6D6F6E2F70726F66736572765F73667470"} 224648
ltmVirtualServStatClientBytesIn{ltmVirtualServStatName="0x2F436F6D6D6F6E2F76735F616374696F6E706C616E6E696E6773657276696365"} 0
```

Any idea if I'm generating my snmp.yml file incorrectly, or should this be handled by the exporter?

**Comment by brian-brazil** [comment]
This looks like an unusual way of indicating an ASCII string in the MIB.

**Comment by cr41gc** [comment]
I just did an snmpwalk and noticed it comes back as a STRING. Going to try replacing OctetString with String.

File: generator/README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Put the extracted mibs in a location NetSNMP can read them from. `$HOME/.snmp/mi
 
 * Cisco: ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz
 * APC: ftp://ftp.apc.com/apc/public/software/pnetmib/mib/419/powernet419.mib
+* F5: https://github.com/librenms/librenms/tree/master/mibs/f5
```
Comment (cr41gc) [inline generator/README.md]: Sadly, F5 doesn't publicly share their MIBs.

**Comment by brian-brazil** [comment]
If the MIBs aren't public then we can't include them I'm afraid, as that'd prevent others from contributing.

**Pull request closed without merging**